x-pack/plugins/cases/public/components/create/flyout/create_case_flyout.tsx

```diff
@@ -5,28 +5,25 @@
  * 2.0.
  */
 
-import React, { memo, useMemo } from 'react';
+import React from 'react';
 import styled, { createGlobalStyle } from 'styled-components';
 import { EuiFlyout, EuiFlyoutHeader, EuiTitle, EuiFlyoutBody } from '@elastic/eui';
 
 import * as i18n from '../translations';
-import { useKibana } from '../../../../../../../../../src/plugins/kibana_react/public';
-import { Case } from '../../../../../../../cases/common';
-import type { TimelinesStartServices } from '../../../../../types';
+import { Case } from '../../../../common';
+import { CreateCaseForm } from '../form';
 
-export interface CreateCaseModalProps {
+export interface CreateCaseFlyoutProps {
   afterCaseCreated?: (theCase: Case) => Promise<void>;
-  onCloseFlyout: () => void;
+  onClose: () => void;
   onSuccess: (theCase: Case) => Promise<void>;
-  useInsertTimeline?: Function;
-  owner: string;
   disableAlerts?: boolean;
 }
 
 const StyledFlyout = styled(EuiFlyout)`
   ${({ theme }) => `
-    z-index: ${theme.eui.euiZLevel5};
-  `}
+      z-index: ${theme.eui.euiZModal};
```

Comment:
Are we sure about it? If I remember correctly there were some bugs regarding multiple modals/flyouts on the same page. For example with the timeline.